src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2306,7 +2311,7 @@
         "conversionSources": {
           "description": "An array of physicalLocation objects which specify the portions of an analysis tool's output that a converter transformed into the result.",
           "type": "array",
-          "minItems": 0,
+          "minItems": 1,
```

Comment:
>minItems [](start = 11, length = 8)

i don;t think this is right. please refer to the issue below for our driving principles. conversion sources is not an object that, if not populated, invalidates the containing object. therefore, in order not to be overly prescriptive, we allow it to be non-null and empty.

[https://github.com/oasis-tcs/sarif-spec/issues/270](https://github.com/oasis-tcs/sarif-spec/issues/270)

Reply:
Thanks! updating

---
In reply to: [276015852](https://github.com/Microsoft/sarif-sdk/pull/1429#discussion_r276015852) [](ancestors = 276015852)